serfclient/connection.py

```diff
@@ -134,3 +134,28 @@ def _counter(self):
     def _error_message(self, exception):
         return "Error %s connecting %s:%s. %s." % \
             (exception.args[0], self.host, self.port, exception.args[1])
+
+    def _decode_addr_key(self, obj_dict):
+        """
+        Callback function to handle the decoding of the 'Addr' field.
+
+        Serf msgpack 'Addr' as an IPv6 address, and the data needs to be unpack
+        using socket.inet_ntop().
+
+        See: https://github.com/KushalP/serfclient-py/issues/20
+
+        :param obj_dict: A dictionary containing the msgpack map.
+        :return: A dictionary with the correct 'Addr' format.
+        """
+        key = b'Addr'
+        if key in obj_dict:
+            ip_addr = socket.inet_ntop(socket.AF_INET6, obj_dict[key])
```

Comment:
What your python you used ? Python3 or python2?
I run line `socket.inet_ntop(socket.AF_INET6, '192.168.0.1')` on python3, but it raise error:
```bash
Traceback (most recent call last):
  File "/home/vcteam/.pycharm_helpers/pydev/pydevd.py", line 1531, in <module>
    globals = debugger.run(setup['file'], None, None, is_module)
  File "/home/xxxx/.pycharm_helpers/pydev/pydevd.py", line 938, in run
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "/home/xxxx/.pycharm_helpers/pydev/_pydev_imps/_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "/home/xxxx/plugins/learn/learn_xxx.py", line 4, in <module>
    print(socket.inet_ntop(socket.AF_INET, '192.168.145.21'))
TypeError: 'str' does not support the buffer interface
```